packages/core/template/default/index.md

```diff
@@ -1,132 +1,75 @@
 <frontmatter>
-  title: Landing Page
+  title: Home Page
   layout: default.md
-  pageNav: 2
-  pageNavTitle: "Chapters of This Page"
+  pageNav: 4
+  pageNavTitle: "Placeholder Topic Pages"
 </frontmatter>
 
 <br>
 
-<div class="bg-primary text-white py-5 mb-4">
+<div class="bg-primary text-white px-2 py-5 mb-4">
   <div class="container">
-    <h1 class="display-4 no-index">Landing Page Title</h1>
-    <p class="lead">A tagline can go here</p>
+    <h1 class="display-5 no-index">Great!<br>You've just initialized a MarkBind site.</h1>
+    <p class="lead">Let's get started...</p>
   </div>
 </div>
 
-# Heading 1
-Some text some text some text some text some text some text some text. **Some text some text some text some text some text ==some text== some text**. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
+---
 
-**A block quote:**
+## What did you just do?
 
-> Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
+You have just initialized a _default_ MarkBind site, which is equipped with a set of core features of a MarkBind site like Site and Page Navigation, as well as some links to our User Guide, to help you get started more conveniently!
```

Comment:
```suggestion
You have just initialized a _default_ MarkBind site! It is equipped with a set of core features, including site and page navigation. Additionally, we have included some handy links to our User Guide to help you get started quickly and easily.
```

Reply:
Thanks for the suggestion! Updated in [fab85db](https://github.com/MarkBind/markbind/pull/2225/commits/fab85db23ed5ba23eacde0a78dc99c45d63d69c7).